src/components/Chips/Chips.tsx

```diff
@@ -26,6 +26,12 @@ export const Chips = (props: TChipsProps): ReactElement => {
 
   const customType = disabled ? 'disabled' : type == 'filled' ? 'inverse' : color
```

Comment:
Maybe it's better to have an enum for all strings or it can be constant if it's a single value